examples/cuda/image_function/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_executable(cuda_example_image_function
```

Comment:
For `.*cu` files we need to use a separate compiler **nvcc**. We have the same problem for CUDA unit tests: we use makefile for compilation. I recommend to don't spend time on CUDA right now and just do not add Cmake files for these 2 project examples.
We could address this problem in a separate issue.

Reply:
That's indeed curious as I checked the `unit_tests/cuda/Makefile` and I wonder how this work because `THIRD_PARTY_DIR := $(LIB_DIR)/thirdparty/multicuda/src` point to a non-existent directory.

Reply:
I think I forgot to remove third party from makefile. The code was there but after a while I merged 2 projects